[GudhUI] ‘class Viewer’ has no member named ‘updateGL’ -- with new version of QGLViewer

With version 2.8.0 of QGLViewer, I get
```
/home/glisse/repos/gudhi/build/2/GudhUI/view/Viewer.cpp: In member function ‘void Viewer::update_GL()’:
/home/glisse/repos/gudhi/build/2/GudhUI/view/Viewer.cpp:34:9: error: ‘class Viewer’ has no member named ‘updateGL’; did you mean ‘update_GL’?
   34 |   this->updateGL();
      |         ^~~~~~~~
      |         update_GL
make[2]: *** [GudhUI/CMakeFiles/GudhUI.dir/build.make:188: GudhUI/CMakeFiles/GudhUI.dir/view/Viewer.cpp.o] Error 1
```

--- a/src/GudhUI/view/Viewer.cpp
+++ b/src/GudhUI/view/Viewer.cpp
@@ -31,7 +31,11 @@ void Viewer::set_bounding_box(const Point_3 & lower_left, const Point_3 & upper_
 }
 
 void Viewer::update_GL() {
+#if QGLVIEWER_VERSION >= 0x020700
+  this->update();
+#else
   this->updateGL();
+#endif
 }
 
 void Viewer::init_scene() {

